packages/field-plugin/src/createFieldPlugin/createFieldPlugin.ts

```diff
@@ -76,8 +79,13 @@ export const createFieldPlugin: CreateFieldPlugin = ({
 
   const cleanupStyleSideEffects = disableDefaultStoryblokStyles()
 
+  // This is basically the `Content` inferred from the `validateContent`.
+  type InferredContent = ReturnType<
```

Comment:
Could you please explain why we are using the Exclude utility class here?

Reply:
So, it's basically..


https://github.com/storyblok/field-plugin/assets/499898/397d47ab-892e-4560-aa44-e56bfa0478dd


does this explain it?

Reply:
that makes total sense, thank you